CS342MemberViews/Sources/DannyPark/Resources/DannyPark.JPG.license

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Great to see that you added license files for your images!

The REUSE check isn't able to find this file is because it has to be the exact same name as the image, except with `.license` at the end. The image is named `DannyPark.jpeg` but this license file is named `DannyPark.JPG.license`. If you change the name of this file to `DannyPark.jpeg.license` the REUSE check will be able to find it!